src/main/java/org/fiximulator/core/FIXimulatorApplication.java

```diff
@@ -572,6 +572,15 @@ public void sendExecution(Execution execution) {
             executionReport.set(new SecurityID(order.getSecurityID()));
             executionReport.set(new IDSource(order.getIdSource()));
         }
+        System.out.println("LiqudityFlag");
+        boolean sendLiquidityFlag = false;
+        try {
+            sendLiquidityFlag = settings.getBool("FIXimulatorSendLiquidityFlag");
+        } catch (Exception ex) {}
+
+        if (sendLiquidityFlag) {
+            executionReport.setString(9882, "5/1"); // Add optional User Defined LiquidityFlag
```

Comment:
Extract string literal to meaningful static variable

Reply:
recommend for `FIXimulatorSendLiquidityFlag` as well